introduction.md

```diff
@@ -1 +1,4 @@
 bryan is a great guy.  he works hard. i like Ruby.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?